### PR TITLE
🌱 Add test for shallow copy and make partial metadata CRD

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vince@prigna.com>

## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/server/apiextensions_test.go [hunk 1]
```diff
@@ -131,3 +131,43 @@ func TestDecorateCRDWithBinding(t *testing.T) {
...
+							Description: "desc",
+							Type:        "bob",
+						},
+					},
+				},
+			},
+		},
+	}
+
+	// shallow := original
```
Comment (ncdc) [inline pkg/server/apiextensions_test.go:154]:
```suggestion
```

Comment (vincepri) [reply pkg/server/apiextensions_test.go]: Fixed, thanks!

**Pull request merged**